packages/spacecat-shared-utils/src/log-wrapper.js

```diff
@@ -11,45 +11,46 @@
  */
 
 /**
- * Wraps the provided function to enhance the logging functionality by including the
- * `jobId` in each log statement.
+ * A wrapper function that enhances logging by appending the `jobId` to all log statements
+ * within the provided context. This allows better traceability and correlation of logs
+ * associated with a specific job.
  *
- * @param {function} fn - The original function that will be wrapped.
- * @returns {function(object, object): Promise<Response>} A new function that, when invoked,
- *  will automatically include the `jobId` in all log statements within the context.
+ * @param {function} fn - The original function to be wrapped.
+ * @returns {function(object, object): Promise<Response>} A wrapped function that includes
+ *  `jobId` in all log statements if `context.contextualLog` is used.
  */
 export function logWrapper(fn) {
   return async (message, context) => {
     const { log } = context;
 
     if (log) {
-      if (!message || !message.jobId) {
-        log.debug('Missing jobId, hence it will not be included in log messages.');
-        context.contextualLog = log;
+      if (message && 'jobId' in message) {
```

Comment:
```suggestion
      if (isObject(message) && 'jobId' in message) {
```

Reply:
ESLint reported a circular dependency issue with the isObject function, so I did not include it.